go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent none
+    stages {
+        stage('TestSQL') {
+            parallel {
+                stage ("update Liquidata/dolt-sql-performance") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        DOLT_VERSION = sh(returnStdout: true, script: "(git rev-parse HEAD | tr -d '\n')")
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        TEST_N_TIMES = 5 // like to pass this from the Jenkins UI?
+                        DOLT_GLOBAL_CONFIG = "/home/jenkins/.dolt/config_global.json"
+                    }
+                    steps {
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("go") {
+                            sh "rm -rf $TMPDIR && mkdir $TMPDIR"
+                            sh "go get -mod=readonly ./..."
+                            sh "go build -mod=readonly -o ../.ci_bin/dolt ./cmd/dolt/."
+                        }
+                        sh "rm -rf dolt-sql-performance && dolt clone Liquidata/dolt-sql-performance"
+                        dir ("go/libraries/doltcore/sqle/logictest/main") {
+                            sh "dolt config --global --add user.name 'LiquidataSystemAccount'"
+                            sh "dolt config --global --add user.email 'liquidatasys@gmail.com'"
+                            sh "rm -rf .dolt && dolt init"
+                            sh "rm -rf $CREDSDIR && mkdir $CREDSDIR"
+                            sh "cat $DOLT_CREDS > /home/jenkins/.dolt/creds/5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is.jwk"
+                            sh "dolt config --global --add user.creds 5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is"
+                            sh "dolt config --global --add metrics.disabled true"
+                            sh "dolt creds use 5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is"
+                            sh "rm -rf temp && mkdir temp"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "go run . run ../../../../../../sqllogictest/test/select1.test > temp/results${i}.log"
```

Comment:
Just using `select1.test` temporarily while I ensure this pipeline works... it will run all tests though when I'm ready to merge this PR.